HarfangLab/harfanglab/ingest/parser.yml

```
@@ -192,6 +195,10 @@ stages:
           action.properties.additional_info.additional_info3: "{{json_event.message.agent.additional_info.additional_info3}}"
           action.properties.additional_info.additional_info4: "{{json_event.message.agent.additional_info.additional_info4}}"
 
+      - set:
+          host.ip: "{{json_event.message.agent.ipaddress}}"
```

`host.ip` must be an array:
```suggestion
          host.ip: ["{{json_event.message.agent.ipaddress}}"]
```